cime_config/config_compsets.xml

```diff
@@ -213,6 +213,12 @@
     <lname>1850_DATM%CPLHIST_CLM50%BGC_SICE_SOCN_MOSART_CISM2%NOEVOLVE_SWAV</lname>
   </compset>
 
+<!-- I CLM50 NWP Compsets -->
+
+  <compset>
+    <alias>I2000Nwp50SpGswpGs</alias>
```

Comment:
Other proposed compset names would be:

1. `I2000Nwp50SpNldas2Gs` - Uses NLDAS2 datm forcing

2. `I2000Nwp50SpNldas2RsGs` - Uses NLDAS2 datm forcing and stub ROF

Reply:
In discussion with @barlage - he would prefer `I2000Ctsm50NwpSpGswpGs`, and that makes sense to me (and similar changes for the two other compsets).